Document inviting a User by Email (#776)
mautic/user-documentation: docs/users_roles/managing_users.rst

* **Language** - Select a language for each User, to improve their experience in Mautic. 

When creating your Users, inform them of the credentials. Mautic doesn't send an Email notifying Users of their login information. Manually informing them is necessary.

.. vale off

> 
> Inviting a User by Email
> ************************
> 
> .. vale on
> 
> Instead of creating a User manually and sharing credentials yourself, you can invite someone by Email. Mautic sends them an invitation link, and the invitee sets up their own account by choosing a username and password. This requires the **Create** permission for Users.
> 
> To invite a User:
> 
> #. Navigate to **Settings** > **Users**.
> 
> #. Click **Invite user** in the top right corner. You can also select **Invite your team** from the profile menu in the top right of the screen.
> 
> #. In the 'Collaborate with your organization' window, enter the **Email address** of the person you want to invite.
> 
> #. Select a **Role** to assign to the invited User.
> 
> #. Click **Send invite**.

Reviewer: There is only 'Invite your team' on UI. We can also add images and descriptive alt text here for clarity. Resize the images to 60%.
 
```suggestion
.. vale off

#. Navigate to **Settings** > **Users]**.
#. Click **Invite your team** at the top right of the table.

   |

   [IMAGE: invite_your_team_button_users_page]

   You can also find and select it from the profile menu.
   
   [IMAGE: invite_your_team_button_profile_menu]
   
   |

#. In the 'Collaborate with your organization' window, enter the **Email address** of the person you want to invite.
#. Select a **Role** to assign to the invited User.
#. Click **Send invite**.

   |

   [IMAGE: invite_your_team]
   
   |

.. vale on
```

Here are the images:

invite_your_team_button_users_page

<img width="2084" height="686" alt="Image" src="https://github.com/user-attachments/assets/c68a009d-98ca-4814-928f-87bc2e8b2177" />

invite_your_team_button_profile_menu

<img width="338" height="443" alt="Image" src="https://github.com/user-attachments/assets/93381480-f6aa-4c3a-9bc5-ca0c0f6cc3d7" />

invite_your_team

<img width="1122" height="1020" alt="Image" src="https://github.com/user-attachments/assets/4166ab46-4637-4a90-9879-53644450b97d" />

> Mautic sends an invitation Email to that address with a link to create an account. The link expires after 48 hours. If you invite the same Email address again, any earlier invitation for that address stops working.

Author: Documented the 48-hour invitation expiration and the re-invite revocation behavior from `UserModel::createInvite()` in [PR #15401](https://github.com/mautic/mautic/pull/15401), which sets the invite expiration to `PT48H` and calls `revokeOutstandingInvites()` before persisting a new invite. The access points (`Invite user` toolbar button and `Invite your team` profile link) come from `User/list.html.twig` and `Menu/profile.html.twig`.

Source: https://github.com/mautic/mautic/pull/15401

.. note::

   You can only invite an Email address that doesn't already belong to an existing User. If the address is already in use, ask that person to sign in with their existing account instead.

.. vale off

Completing the invitation
=========================

.. vale on

> When the invited person opens the link in the Email, Mautic shows a registration form where they finish setting up their account. The form pre-fills the Email address and Role from the invitation, and the invitee provides:

Author: The registration form fields (`username`, `firstName`, `lastName`, `plainPassword`, `locale`) and the `Create Account` button come from `UserInviteRegistrationType.php`; the pre-filled Email and Role come from `User::createFromInvite()`, and the redirect to login on success comes from `PublicController::inviteAction()` in [PR #15401](https://github.com/mautic/mautic/pull/15401).

Source: https://github.com/mautic/mautic/pull/15401

> When the invited person opens the link in the Email, Mautic shows a registration form where they finish setting up their account. The form pre-fills the Email address and Role from the invitation, and the invitee provides:

Reviewer: The word 'form' here are referring to generic form, not Mautic's feature.

```suggestion
.. vale off

When the invited person opens the link in the Email, Mautic shows a registration form where they finish setting up their account. The form pre-fills the Email address and Role from the invitation, and the invitee provides:

.. vale on
```

* **Username** - The name they use to log in.

* **First name** and **Last name** - The invitee's name.

* **Password** and **Confirm Password** - Their login password. Passwords must be at least six characters in length. Use a combination of upper and lower case letters, special characters, and numbers.

* **Language** - The language for their Mautic experience, or the system default.

> After the invitee clicks **Create Account**, Mautic creates the User and redirects them to the login page, where they can sign in with their new credentials.

Reviewer: The word 'page' here is generic.

```suggestion
.. vale off

After the invitee clicks **Create Account**, Mautic creates the User and redirects them to the login page, where they can sign in with their new credentials.

.. vale on
```

* **Language** - Select a language for each User, to improve their experience in Mautic. 

When creating your Users, inform them of the credentials. Mautic doesn't send an Email notifying Users of their login information. Manually informing them is necessary.

.. vale off

Inviting a User by Email
************************

.. vale on

Instead of creating a User manually and sharing credentials yourself, you can invite someone by Email. Mautic sends them an invitation link, and the invitee sets up their own account by choosing a username and password. This requires the **Create** permission for Users.

To invite a User:

.. vale off

#. Navigate to **Settings** > **Users**.
#. Click **Invite your team** at the top right of the table.

   |

   .. image:: images/invite_your_team_button_users_page.png
     :alt: Users page with the Invite your team button highlighted at the top right of the table.

   You can also find and select it from the profile menu.

   .. image:: images/invite_your_team_button_profile_menu.png

>      :width: 400

Reviewer: This is still too big. Perhaps because the height is bigger than the width? Maybe we should target the height instead of the width.

     :alt: Profile menu in the top right of the screen with the Invite your team option highlighted.

   |

#. In the 'Collaborate with your organization' window, enter the **Email address** of the person you want to invite.
#. Select a **Role** to assign to the invited User.
#. Click **Send invite**.

   |

   .. image:: images/invite_your_team.png

>      :alt: Collaborate with your organization window showing the Email address field, the Role selector, and the Send invite button.

Reviewer: Add `:width: 600`.

   |

.. vale on

Mautic sends an invitation Email to that address with a link to create an account. The link expires after 48 hours. If you invite the same Email address again, any earlier invitation for that address stops working.

.. note::

   You can only invite an Email address that doesn't already belong to an existing User. If the address is already in use, ask that person to sign in with their existing account instead.

.. vale off

Completing the invitation
=========================

When the invited person opens the link in the Email, Mautic shows a registration form where they finish setting up their account. The form pre-fills the Email address and Role from the invitation, and the invitee provides:

.. vale on

* **Username** - The name they use to log in.

* **First name** and **Last name** - The invitee's name.

> 
> * **Password** and **Confirm Password** - Their login password. Passwords must be at least six characters in length. Use a combination of upper and lower case letters, special characters, and numbers.
> 
> * **Language** - The language for their Mautic experience, or the system default.

Reviewer: ```suggestion
* **Username** - The name they use to log in.
* **First name** and **Last name** - The invitee's name.
* **Password** and **Confirm Password** - Their login password. Passwords must be at least six characters in length. Use a combination of upper and lower case letters, special characters, and numbers.
* **Language** - The language for their Mautic experience, or the system default.
```

* **Language** - Select a language for each User, to improve their experience in Mautic. 

When creating your Users, inform them of the credentials. Mautic doesn't send an Email notifying Users of their login information. Manually informing them is necessary.

.. vale off

Inviting a User by Email
************************

.. vale on

Instead of creating a User manually and sharing credentials yourself, you can invite someone by Email. Mautic sends them an invitation link, and the invitee sets up their own account by choosing a username and password. This requires the **Create** permission for Users.

To invite a User:

.. vale off

#. Navigate to **Settings** > **Users**.
#. Click **Invite your team** at the top right of the table.

   |

   .. image:: images/invite_your_team_button_users_page.png

>      :width: 60%

Reviewer: Remove this.

```suggestion
```

     :alt: Users page with the Invite your team button highlighted at the top right of the table.

   You can also find and select it from the profile menu.

   .. image:: images/invite_your_team_button_profile_menu.png

>      :width: 60%

Reviewer: This image looks quite gigantic. The 60% isn't work here. Should we use pixel instead? Maybe something like `width: 400`?

     :alt: Profile menu in the top right of the screen with the Invite your team option highlighted.

   |

#. In the 'Collaborate with your organization' window, enter the **Email address** of the person you want to invite.
#. Select a **Role** to assign to the invited User.
#. Click **Send invite**.

   |

   .. image:: images/invite_your_team.png

>      :width: 60%

Reviewer: Remove this.

```suggestion
```
